truffle/src/main/ruby/core/rubinius/api/shims/tuple.rb

```diff
@@ -28,6 +22,48 @@ def copy_from(other, start, length, dest)
 
 class Array
 
+  def copy_from(other, start, length, dest)
+    # Rubinius.primitive :tuple_copy_from
```

Comment:
Is there any reason to not implement the primitive?  At the very least, we should probably add it and just return `null` so it falls back to the Ruby code.